packages/server/lib/controllers/oauth.controller.ts

```diff
@@ -245,7 +245,7 @@ class OAuthController {
             }
 
             // certain providers need the credentials to be specified in the config
-            if (overrideCredentials) {
+            if (overrideCredentials && (overrideCredentials['oauth_client_id_override'] || overrideCredentials['oauth_client_secret_override'])) {
```

Comment:
not sure I follow why we are adding this condition

Reply:
With a github oauth app credentials are actually passed in so in this scenario override was a false positive.